### ISSUE TITLE
Languages

### ISSUE THREAD

**Issue by unleashy** [conworkshop/conworkshop#1]
- [ ] Add phonology manager
  - [ ] CRUD for phonemes
  - [ ] Ortho for phonemes -- probably good to revamp this area?
- [ ] Add dictionary manager
  - [ ] Quick word addition?
- [ ] Words
  - [ ] Word relationships, synonyms, homonyms
  - [ ] Roots
  - [ ] Custom classes
  - [ ] Etymology and other fields

**Comment by m5ka** [comment]
I've talked with some other people (an avid CWSer and a hobby phoneticist) and I think the **phono manager** definitely needs probably one of the biggest overhauls of this whole section.

Proposal:
- The edit ortho and edit phono pages merge into the main phono page, where all the editing is done through the smart popup dialogues (which would, of course, be vastly improved compared to the cws1 ones)
- The add phoneme page is also done with a smart popup dialogue, where a phoneme is clicked and the popup allows its customisation (e.g diacritics/qualities, orthography, allophony, notes) before it's saved

**Comment by several-lizards** [comment]
@bheara Will the popup edit ortho/phono pages still be list-form? Because it's hell to edit all that stuff one phoneme/grapheme at a time

**Comment by m5ka** [comment]
@several-lizards that's true actually - perhaps both could be kept then. so the phono edit page has the interactive phono table with the popups when you click a phoneme but theres still a link to a mass edit page where the quality etc ANd ortho of all the phonemes can be fiddled with
